modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayBuild.macro

```diff
@@ -0,0 +1,69 @@
+definition {
+
+	macro addBuild {
+		Click(locator1 = "Testray#ADD_PROJECT");
+
+		Type(
+			locator1 = "TestrayManage#NEW_OPTION_NAME",
+			value1 = "${fieldName}");
+
+		Select(
+			locator1 = "TestrayBuild#ROUTINE_ID",
+			value1 = "${routine}");
+
+		Select(
+			locator1 = "TestrayBuild#PRODUCT_VERSION",
+			value1 = "${productVersion}");
+
+		Type(
+			locator1 = "TestrayBuild#GIT_HASH_FIELD",
+			value1 = "${gitHash}");
+
+		Type(
+			locator1 = "TestrayCases#FIELD_DESCRIPTION",
+			value1 = "${Description}");
```

Comment:
Hi @joelrocha68, this code here is the same as the code from the editBuild macro (lines 46 to 64). You can create a new macro just to fill testray build values in order to avoid duplicate code.